compliance_checker/cf/cf.py

```diff
@@ -139,6 +139,109 @@ def setup(self, ds):
         self._find_cf_standard_name_table(ds)
         self._find_geophysical_vars(ds)
 
+    def _check_conventions_version(self, ds):
```

Comment:
Was this intended to be part of this PR?

Reply:
It was not... I'll have to investigate what happened here.